### ISSUE TITLE
Idea: Post module freeze notification (new_plugin)

### ISSUE THREAD

**Issue by gundalow** [ansible/ansibullbot#412]
Something I thought about, and may be done by a human rather than the bot, though this seems like a sensible place to record it.

Currently we are in feature freeze, and will be until we branch `stable-2.3`, and which point `devel` becomes 2.4. Then we can start accepting new modules into `devel` again.

Any new modules will need updating to include `version_added: "2.4`
Also I'm aiming by RC1 to have must stricter `validate-module` checks in (WIP https://github.com/ansible/ansible/pull/22353) so we can raise the bar for 2.4.

Therefore I'm wondering if every PR with `label:new_plugin` (maybe since `stable-2.2` was branched) should get a messaging.

Wording may need a bit more polishing, and maybe extra requirements added.

> Now that Ansible 2.3 has reached RC1[1] the `devel` branch is now versioned as `2.4` and can accept new modules again.
> 
> There are just a few bits of house keeping to be done:
> 
> 1) Rebase your PR http://docs.ansible.com/ansible/dev_guide/developing_rebasing.html
> 2) Update `version_added` to be `version_added: "2.4"
> 3) Review your module against the updated guidelines http://docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html and update as needed.
> 
> As a reminder, any code changes reset the number of "shipits" to zero, so once the above is complete add a message with the test `ready_for_review` to make it clear that you wish people to look at the code.
> 
> Once your new module gets two shipits (without any code changes in between) it will be marked for inclusion in the next release [2].
> 
> 
> [1] Link to google-devel post announcing 2.3 RC1 - Help testing the Ansible 2.3 release candidates is much appreciated
> [2] https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md#new-modules

@mattclay FYI, would be good to get your thoughts on this

**Comment by mattclay** [comment]
The [filter](https://github.com/ansible/ansible/pulls?utf8=%E2%9C%93&q=is%3Apr%20is%3Aopen%20label%3Anew_module) should probably be on `new_module` instead, for which there are currently 169 open PRs.

I'd drop the first step from the instructions. There isn't anything to be gained from rebasing a PR for a new module, unless you're going to run tests locally.

Since we've been adding (and are continuing to add) additional sanity checks to CI, we may want to re-run CI on all of these PRs and let the bot comment on them instead. At a minimum it will point out the version_added requirement. That way the contributor can see all the bot feedback in one pass, instead of seeing the CI failures after they push their update to version_added.

**Comment by gundalow** [comment]
Updates to `validate-modules` are now live https://github.com/ansible/ansible/pull/22353 has been merged

**Comment by gundalow** [comment]
I'm not sure if this is a good thing. Notifying people to do stuff without someone to merge isn't good

**Comment by mkrizek** [comment]
With the collections and the core team not accepting new plugins into ansible/ansible this is no longer needed. I am going to go ahead and close this.
